bin/aws_bash_completer

```diff
@@ -1,6 +0,0 @@
-# Typically that would be added under one of the following paths:
```

Comment:
For the bash and zsh completers, I was thinking it may be worth keeping it in the repository and labeled as a script for now in the `setup.cfg`. For Linux distros/Homebrew, they automatically take these files and configure as part of the install to automatically get command completion. For the purpose of our source distribution though, we can just not copy it into the bin directory.